### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by jeeyun**
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/jeeyun/codereviewhub/pull/1%23issuecomment-257436486%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222016-10-31T22%3A14%3A13Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1827742%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jeeyun%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%3Agun%3A%22%2C%20%22created_at%22%3A%20%222016-10-31T23%3A01%3A06Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1827742%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jeeyun%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%205454754c82759fdedfa0175e8bd35ed452166cb3%20README.md%202%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/jeeyun/codereviewhub/pull/1%23discussion_r85843249%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22can%20you%20make%20it%20lowercase%20%3F%22%2C%20%22created_at%22%3A%20%222016-10-31T22%3A04%3A50Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/23178397%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/justjeeyun%22%7D%7D%2C%20%7B%22body%22%3A%20%22ok%22%2C%20%22created_at%22%3A%20%222016-10-31T22%3A07%3A14Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1827742%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jeeyun%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222016-10-31T22%3A14%3A26Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1827742%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jeeyun%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20README.md%3AL1-3%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/jeeyun/codereviewhub/pull/1%23issuecomment-257436486%22%2C%20%22https%3A//github.com/jeeyun/codereviewhub/pull/1%23discussion_r85843249%22%2C%20%22https%3A//github.com/jeeyun/codereviewhub/pull/1%23discussion_r85843584%22%2C%20%22https%3A//github.com/jeeyun/codereviewhub/pull/1%23discussion_r85844613%22%2C%20%22https%3A//github.com/jeeyun/codereviewhub/pull/1%23issuecomment-257445380%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/jeeyun/codereviewhub/pull/1#issuecomment-257436486'>General Comment</a></b>
- <a href='https://github.com/jeeyun'></a> :shipit:
- <a href='https://github.com/jeeyun'></a> :shipit::gun:
- [x] <a href='#crh-comment-Pull 5454754c82759fdedfa0175e8bd35ed452166cb3 README.md 2'></a> &nbsp;<b><a href='https://github.com/jeeyun/codereviewhub/pull/1#discussion_r85843249'>File: README.md:L1-3</a></b>
- <a href='https://github.com/justjeeyun'></a> can you make it lowercase ?
- <a href='https://github.com/jeeyun'></a> ok

<a href='https://www.codereviewhub.com/jeeyun/codereviewhub/pull/1?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/jeeyun/codereviewhub/pull/1?approve=1'></a>&nbsp;<a href='https://github.com/jeeyun/codereviewhub/pull/1'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by jeeyun** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/jeeyun/codereviewhub/pull/1'>Refresh</a>.<a href='#crh-update-none'></a>

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# codereviewhub
+YARS
```
Comment (justjeeyun) [inline README.md]: can you make it lowercase ?

Comment (jeeyun) [inline README.md]: ok

**Comment by jeeyun** [comment]
:shipit:

File: README.md (hunk 1 above)
Comment (jeeyun) [inline README.md]: :+1:

**Comment by jeeyun** [comment]
:shipit::gun:

**Pull request merged**